styles.css

```diff
@@ -12,6 +12,15 @@ body {
   text-align: center;
 }
 
+.button {
+  background: #134d71;
```

Comment:
Not a big deal but make this `background-color to be more semantic

Reply:
OK- Thank you for pointing that out. I will make the changes.